### PR TITLE
Fix footerlink to guide

### LINKED ISSUE #35: 🐛 [Bug report]: Footer Guide Link is broken
### Descrcibe the bug?

The guides link in the footer is broken and redirects to https://react-play-docs.netlify.app/docs/intro ( this route does not exist).

### Steps to reproduce the bug?

image

### Expected behavior

It should redirect to `https://react-play-docs.netlify.app/` instead if `https://react-play-docs.netlify.app/docs/intro`

### Desktop (Please provide your system information)

Browser [e.g. chrome]
Version [e.g. 113+]

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ArslanYM**
Fixed #35

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -103,6 +104,10 @@ const config = {
                 label: 'Twitter',
                 href: TWITTER_URL,
               },
+              {
+                label: 'LinkedIn',
+                href: LINKEDIN_URL,
+              },
```
Comment (Sachin-chaurasiya) [inline docusaurus.config.js:110]: @ArslanYM, You have these changes in PR #38, can you make sure PR has changes related to the mentioned issue?

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (ArslanYM) [reply docusaurus.config.js:110]: i dont understand what youre asking me to do?

Comment (Sachin-chaurasiya) [reply docusaurus.config.js:110]:
```suggestion
              {
                label: 'LinkedIn',
                href: LINKEDIN_URL,
              },
```

You have already made these changes in PR #38. so why we are again include these changes in this PR?

File: docusaurus.config.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const BLOG_URL = 'https://blog.reactplay.io/';
 const GITHUB_URL = 'https://github.com/reactplay/docs/';
 const DISCORD_URL = 'https://discord.gg/vrTxWUP8Am';
 const TWITTER_URL = 'https://twitter.com/reactplayio';
+const LINKEDIN_URL = 'https://www.linkedin.com/company/reactplay/';
```
Comment (Sachin-chaurasiya) [inline docusaurus.config.js:18]: @ArslanYM , Can you remove this as well?

**Review by Sachin-chaurasiya — APPROVED** [review]
Great work @ArslanYM  🙌🏻,
Thanks for the contributions ✨

**Review by atapas — APPROVED** [review]

**Pull request merged**